packages/wallet/src/redux/__tests__/state-helpers.ts

```diff
@@ -31,7 +31,14 @@ export const participants: ChannelParticipant[] = [
   {signingAddress: bsAddress}
 ];
 
-export const libraryAddress = "0x" + "1".repeat(40);
+// TrivialApp.sol
+export const trivialAppBytecode =
+  "0x608060405234801561001057600080fd5b506102e8806100206000396000f3fe608060405234801561001057600080fd5b506004361061002b5760003560e01c806395127f9914610030575b600080fd5b61004a60048036036100459190810190610153565b610060565b60405161005791906101f5565b60405180910390f35b600060019050949350505050565b600082601f83011261007f57600080fd5b813561009261008d8261023d565b610210565b915080825260208301602083018583830111156100ae57600080fd5b6100b983828461027f565b50505092915050565b6000604082840312156100d457600080fd5b6100de6040610210565b9050600082013567ffffffffffffffff8111156100fa57600080fd5b6101068482850161006e565b600083015250602082013567ffffffffffffffff81111561012657600080fd5b6101328482850161006e565b60208301525092915050565b60008135905061014d8161028e565b92915050565b6000806000806080858703121561016957600080fd5b600085013567ffffffffffffffff81111561018357600080fd5b61018f878288016100c2565b945050602085013567ffffffffffffffff8111156101ac57600080fd5b6101b8878288016100c2565b93505060406101c98782880161013e565b92505060606101da8782880161013e565b91505092959194509250565b6101ef81610269565b82525050565b600060208201905061020a60008301846101e6565b92915050565b6000604051905081810181811067ffffffffffffffff8211171561023357600080fd5b8060405250919050565b600067ffffffffffffffff82111561025457600080fd5b601f19601f8301169050602081019050919050565b60008115159050919050565b6000819050919050565b82818337600083830152505050565b61029781610275565b81146102a257600080fd5b5056fea365627a7a72315820caf8688ba5ffb73a0588c13ff6974f5a3ab9edeaeb4246c1b6bcbddb039edd9f6c6578706572696d656e74616cf564736f6c634300050b0040";
+
+export const consensusAppBytecode =
```

Comment:
Could we move this to `constants.ts` for now (and add an issue to populate on load)?